resources/dev_resources/EndpointResourcesList.json

```diff
@@ -2,7 +2,7 @@
     {
         "EndpointName":"Epic",
```

Comment:
I think we'd want to use the EndpointName "FHIR" here so it's processed as a Bundle of FHIR Endpoints